cpp/include/raft/neighbors/detail/cagra/search_multi_cta.cuh

```diff
@@ -175,6 +175,16 @@ struct search : public search_plan_impl<DATA_T, INDEX_T, DISTANCE_T> {
     topk_workspace.resize(topk_workspace_size, resource::get_cuda_stream(res));
   }
 
+  virtual void check(const uint32_t topk)
```

Comment:
The check for `topk <= itopk_size` is not needed anymore for multi-CTA?

Reply:
```suggestion
  void check(const uint32_t topk) override
```
`virtual` is not needed here, but `override` can be used to ensure that the `check` function is virtual.

Reply:
> The check for topk <= itopk_size is not needed anymore for multi-CTA?

Not needed. In the case of multi-CTA, `itopk_size` is overwritten to 32 and `(actual) itopk_size / 32` CTAs are launched to search approximate neighbors of the same query.
Therefore, https://github.com/enp1s0/raft/blob/496689eb042adf80ab3bbbba8cc546ef573d5f11/cpp/include/raft/neighbors/detail/cagra/search_multi_cta.cuh#L180-L185 is enough.